src/coreclr/gc/env/gcenv.base.h

```diff
@@ -485,7 +485,7 @@ class Object;
 class ArrayBase;
 
 // Various types used to refer to object references or handles. This will get more complex if we decide
-// Redhawk wants to wrap object references in the debug build.
+// NativeAOT wants to wrap object references in the debug build.
```

Comment:
This comment doesn't look useful. I would just delete.

Reply:
```suggestion
```